nncf/torch/dynamic_graph/patch_pytorch.py

```diff
@@ -201,7 +201,12 @@ def torch_jit_script_wrapper(*args, **kwargs):
             frames_up = bound_args.get("_frames_up", 0)
             rcb = createResolutionCallbackFromFrame(frames_up + 1)
             kwargs["_rcb"] = rcb
-        retval = _ORIG_JIT_SCRIPT(*args, **kwargs)
+        try:
+            retval = _ORIG_JIT_SCRIPT(*args, **kwargs)
+        except Exception as e:
```

Comment:
Could you please add a small comment here for when this is needed? May be similar to the PR description

Reply:
Commented in the contextmanager